pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -532,10 +541,10 @@ func (pl *InterPodAffinity) getMatchingAntiAffinityTopologyPairsOfPods(pod *v1.P
 // satisfiesPodsAffinityAntiAffinity checks if scheduling the pod onto this node would break any term of this pod.
 func (pl *InterPodAffinity) satisfiesPodsAffinityAntiAffinity(pod *v1.Pod,
```

Comment:
please add a comment documenting the two flags the function returns.

Reply:
added.